app/keygen/README.md

```diff
@@ -9,7 +9,7 @@
 
 **VSecM Keygen** is a utility application that generates VSecM Safe’s 
 bootstrapping keys if you want an extra level of security and control the
-creation of the master key yourself (*instead of letting VSecM Safe generate
+creation of the **root key** yourself (*instead of letting VSecM Safe generate
```

Comment:
Instead of *master key*, or *master secret*, we use *root key*, and *root secret*.